backend/inspirehep/orcid/ext.py

```diff
@@ -34,28 +30,4 @@ def init_config(self, app):
         )
 
     def configure_appmetrics(self, app):
```

Comment:
are we using this function anywhere?

Reply:
now it would be useless, we used it in inspire orcid app https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/orcid/ext.py

Reply:
let's remove it :)